src/Symfony/Component/Notifier/Bridge/LinkedIn/LinkedInOptions.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LinkedIn;
+
+use Symfony\Component\Notifier\Bridge\LinkedIn\Share\AuthorShare;
+use Symfony\Component\Notifier\Bridge\LinkedIn\Share\LifecycleStateShare;
+use Symfony\Component\Notifier\Bridge\LinkedIn\Share\ShareContentShare;
+use Symfony\Component\Notifier\Bridge\LinkedIn\Share\VisibilityShare;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @author Smaïne Milianni <smaine.milianni@gmail.com>
+ *
+ * @experimental in 5.2
+ */
+final class LinkedInOptions implements MessageOptionsInterface
+{
+    private $options = [];
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    public function toArray(): array
+    {
+        return $this->options;
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return null;
+    }
+
+    public static function fromNotification(Notification $notification): self
+    {
+        $options = new self();
+        $options->specificContent((new ShareContentShare($notification->getSubject())));
```

Comment:
extra `()` can be removed (same 3 more times below)

Reply:
fixed